README.md

```diff
@@ -1,72 +1,59 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# HTML/CSS & JavaScript capstone project - D'GEES InfoTech Global Systems Summit page
 
+  > This Project is about creating a responsive webpage
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- HTML
+- CSS
+- Javascript
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo 
 
+[Live Demo Link](https://drive.google.com/file/d/13Km0UwaQjy5rvKNykDjiykMde0BPqfpX/view?)
 
```

Comment:
Great job recording a presentation video describing your project. 👏🏾 

- However, the live demo link should be the URL of your deployed website. Please consider making use of GitHub Pages to deploy the web pages.

An easy way to deploy your application, is on your repository go to > Settings

![image](https://user-images.githubusercontent.com/38088818/176727809-0db7ad5a-1624-41a7-8933-918756d53296.png)

On the left panel, go to > Pages

![image](https://user-images.githubusercontent.com/38088818/176728044-fca4905f-b8f6-4a55-9ade-8968a56a94d3.png)

Select the branch you want to deploy, in your case `capstone-project`, and save. However, when you are done with the development stage and want to merge everything to the final branch `main`. You might want to change the deployed branch to `main` and save.

![image](https://user-images.githubusercontent.com/38088818/176729004-9fd57501-a071-4de3-8543-001ac3ec1633.png)

Wait for Github to build your application, and you will be provided with the URL for your website.

![image](https://user-images.githubusercontent.com/38088818/176729376-246c94ba-9400-4a4b-be25-4c3dc72cd2d2.png)

- Kindly use the published URL as your Live Demo Link, then create a different link for the presentation video.